docs/monitoring.md

```diff
@@ -253,10 +273,24 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>Summary metrics of all tasks in the given stage attempt</td>
   </tr>
   <tr>
+    <td></td>
+    <td><code>?quantiles</code> summarize the metrics with the given quantiles.
+    Example: <br><code>?quantiles=0.01,0.5,0.99</code></td>
```

Comment:
Whatever works, I thought HTML was technically supposed to act like XML and use the self-closing syntax for tags with no body.